nextflow.config

```diff
@@ -55,14 +56,15 @@ params {
   force_option = false
 
   // Workflow flags
-  // TODO nf-core: Specify your pipeline's command line flags
   outdir = './results'
   publish_dir_mode = 'copy'
 
   // Boilerplate options
   name = false
+  multiqc_config = false
```

Comment:
so this multiqc stuff always present?

Reply:
it is part of the nf-core template, I guess I could remove it since I don't use it though @ggabernet do you know if I can remove it?

Reply:
you can remove it if you do not have the multiqc process in your pipeline